db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema[7.0].define(version: 2022_05_31_183553) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```

Comment:
I'm quite certain this file was an accidental commit?

Reply:
Thanks, Mac, I fixed that.